File: Baekjoon/Class1/24.11720.js
const inputFile = process.platform === "linux" ? "/dev/stdin" : "./input.txt";
const input = require("fs").readFileSync(inputFile, "utf8").trim().split("\n");

const nums = input[1];
let total = 0;
for(const num of nums)
    total += Number(num);
console.log(total);

/*
const nums = input[1];
let total = nums.split("").reduce((acc, cur) => acc + Number(cur), 0);
console.log(total);
*/

File: Baekjoon/Class1/24.11720.py
import sys
input = sys.stdin.readline

_ = int(input())
nums = list(map(int, input().rstrip()))
print(sum(nums))

'''
_ = int(input())
nums = list(map(int, input().rstrip()))
total = 0
for num in nums:
    total += num
print(total)
'''